{"messages":[{"error_id":"Release","references":[{"column":9,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/025-MPI-static-mult-win-local-yes.c","line":45},{"column":9,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/025-MPI-static-mult-win-local-yes.c","line":42}],"text":"Found call to MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/025-MPI-static-mult-win-local-yes.c:45:9 which is in conflict with MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/025-MPI-static-mult-win-local-yes.c:42:9 before release","type":"Local Data Race - Local write by call"}]}
